system/api/create.go

```diff
@@ -242,3 +219,86 @@ func createContentHandler(res http.ResponseWriter, req *http.Request) {
 	}
 
 }
+
+func createContentHandlerJSON(res http.ResponseWriter, req *http.Request) (*url.Values, error) {
+	body, err := ioutil.ReadAll(req.Body)
```

Comment:
In fact a second read return empty `[]byte`, there need to be a better way of doing this

Reply:
I updated that 8129842